content/_data/changelogs/weekly.yml

```diff
@@ -15010,6 +15010,45 @@
   # pull: 6407 (PR title: Bump jna from 5.10.0 to 5.11.0)
   # pull: 6410 (PR title: Update SCM URLs)
 
+  - version: '2.342'
+    date: 2022-04-03
+    changes:
+      - type: rfe
+        category: rfe
+        pull: 6422
+        authors:
+          - basil
+        pr_title: Upgrade Spring Framework from 5.3.16 to 5.3.18
+        message: |-
+          Upgrade [Spring Framework](https://spring.io/projects/spring-framework) 5.3.16 (released on February 17, 2022) to [5.3.18](https://github.com/spring-projects/spring-framework/releases/tag/v5.3.18) (released on March 31, 2022). This release of Spring Framework addresses the security vulnerability [CVE-2022-22965](https://tanzu.vmware.com/security/cve-2022-22965).
```

Comment:
Needs revision after the release so that the links can be moved into a references section